pkgs/development/embedded/bootcommander/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib, stdenv, fetchgit, cmake, libopenblt }:
+
+stdenv.mkDerivation rec {
+
+  pname = "bootcommander";
+  version = "011500";
+
+  src = fetchgit {
```

Comment:
Please use fetchFromGitHub

Reply:
the github repo is a mirror

Reply:
does fetchFromGithub support partial clone? because the repo is too big ad fool of useless examples

Reply:
it does support sparse checkout